### PR TITLE
Add support to Rails 7

### PR DESCRIPTION
**Pull request by fabiolnm**
I added these minimal changes to enable using `minitest-rails` in a project I just started.
Are there other changes I should implement n order to make the PR mergeable into the main repository?

### CODE REVIEW

**Comment by blowmage** [comment]
Wonderful, thanks! I will try to get master moved to Rails 7 this weekend. Thank you for contributing!

**Comment by daande** [comment]
@blowmage can you get this merged and make a release as Rails 7 was officially released yesterday  https://rubyonrails.org/2021/12/15/Rails-7-fulfilling-a-vision

File: gemfiles/rails-7.0.0.gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "http://rubygems.org"
 
 gem "minitest-rails", path: "../"
-gem "rails", "6.1.0"
+gem "rails", "7.0.0.rc1"
```
Comment (daande) [inline gemfiles/rails-7.0.0.gemfile:4]:
```suggestion
gem "rails", "7.0.0"
```

File: minitest-rails.gemspec [hunk 2]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.5.0"
 
   gem.add_dependency "minitest", "~> 5.10"
-  gem.add_dependency "railties", "~> 6.1.0"
+  gem.add_dependency "railties", "7.0.0.rc1"
```
Comment (daande) [inline minitest-rails.gemspec:21]:
```suggestion
  gem.add_dependency "railties", "7.0.0"
```

**Comment by haseebab** [comment]
Thanks everyone! Really excited to start migrating some of the projects over to Rails 7!

**Pull request merged**